### ISSUE TITLE
Feature request: Add support to control text size (independently from system settings)

### ISSUE THREAD

**Issue by tinybot-ca** [Ranchero-Software/NetNewsWire#2566]
Would love the ability to set text/font size independently from the system settings.

**Comment by vincode-io** [comment]
Could you be more specific?  Just the article view or the whole app?  Which platform do you use most?

**Comment by vincode-io** [comment]
Oops. I didn't mean to close this.  I just hit the wrong button.

**Comment by tinybot-ca** [comment]
> Could you be more specific? Just the article view or the whole app? Which platform do you use most?

For the article view. I'm on iOS (iPhone and iPad).

**Comment by invariant** [comment]
May I add  my +1 to this, but or the Mac,
Specifically, I would like to increase the text size in the reading pane on Mac. At present it's just a little too small for comfortable relaxed reading.

My vision is not great and I already have the text size set quite large in Display settings, which is OK for detailed close-up work on the Mac, but when I read I like to sit back in my chair, which increases my distance from the screen. 

So I would really appreciate the ability to set a much large text size (at least double current size) for reading articles. (No need to increase size of controls erc.)

Many thanks for your considerarion.

**Comment by vincode-io** [comment]
@invariant you are in luck.  We implemented this for NetNewsWire 6.0 for Mac in #42. 

We haven't implemented it for iOS because iOS uses Dynamic Type.  We figure if you are having trouble reading an article on an iOS device, you have increased the Dynamic Type text size since you will be having trouble reading other things on the device.

We are very reticent to add new settings on NetNewsWire.  Too many and they can become overwhelming and too complicated.  We haven't decided yet if this will be an iOS feature or not.

**Comment by tinybot-ca** [comment]
> We are very reticent to add new settings on NetNewsWire. Too many and they can become overwhelming and too complicated. We haven't decided yet if this will be an iOS feature or not.

I totally understand wanting to avoid feature creep. 

But this is one of those, I would almost describe it as a table stakes kind of feature for an RSS reading app. 

I like the default text size for general iOS usage just fine -- but when I go to load up my fave RSS reader, there are many times I just want to kick back, relax, and really enjoy and savor the experience of reading content that I know has been accumulating from my favorite writers and sites. And when I do that, I tend to have the iPhone or iPad further away from me (imagine taking a moment to relax on the couch, taking a deep breath, stretching out into a more comfortable posture).

Contrast this with keeping up with texts from colleagues, emails from the boss, the latest project work chatter on Slack --- almost always I find that I'm more tense, and tend to keep my iPhone in particular much closer to my face as a result. 

Anyway, I hope you folks will take into consideration. For now, I'm wobbling back and forth between Reeder and NNW, but am really hoping to make NNW my permanent long-term RSS home. I would happily pay for this.

**Comment by pablosproject** [comment]
I will like to add a +1 on this and, if possible, will try to implement the font adjusting in the iOS app if this is something that will be interesting for someone.
I am an avid user of the "reading mode" in safari, where you can control the font size and colors of the text you are reading. Often I am reading the news at night with tired eyes, and increase the text size makes it really easy for me and permits me to read longer (like I do on other devices, like the Kindle).

**Comment by adinklotz** [comment]
FYI iOS actually has system-level support for per-app text size using the text size Control Center control

![image](https://user-images.githubusercontent.com/16441122/162580809-e496d687-7f13-4d4e-a849-17fed932529e.jpeg)

**Comment by brentsimmons** [comment]
I had no idea! I wonder why this doesn’t get passed through the regular dynamic text API — you‘d think we’d get support for this automatically. Well, we’ll look into it and see what the deal is.

**Comment by blackketter** [comment]
Works automatically for me!

**Comment by adinklotz** [comment]
Yeah I don't know if there's anything Brent needs to do, I think it works automatically for anyone using native text APIs. NNW may or may not also want an in-app text size option, but for now at least this should do the job

**Comment by brentsimmons** [comment]
Great! Maybe there’s nothing we need to do — other than to tell people about this solution.

**Comment by hanche** [comment]
> other than to tell people about this solution

Indeed, please do! Perhaps add it to the [NetNewsWire 6.1 for iPhone and iPad Help](https://netnewswire.com/help/ios/6.1/en/) somewhere, perhaps in a separate “Tips and Hints” section? And maybe to the FAQ as well.

This is a fairly recent and little known addition to iOS/iPadOS, I believe. Also, since the Mac application does have the option to alter the font size, it is only natural to expect it to be there in the mobile app too. I certainly spent a ridiculous amount of time researching this before it occured to me to come here. And most users probably don't even know about github.

As a bonus, adding information for users should be quick and easy, requires no testing, and lets you close this issue. Win-win-win.

**Comment by brentsimmons** [comment]
This is a NetNewsWire documentation bug — we need to let people know how to do this.
